src/components/MultiButton.tsx

```diff
@@ -110,7 +112,7 @@ export const MultiButton = ({
       {!hideSecondaryButton && (
         <>
           <Spacer data-disabled={disabled} />
-          <Root>
+          <Root modal={false}>
```

Comment:
Dette er for å at scrollbaren ikke skal forsvinne når dropdownen åpnes, dette gjorde at alt innholdet sideforskyves i footer